data/system_outputs/fb15k-237/no-user-defined.json

```diff
@@ -0,0 +1,142 @@
+[
```

Comment:
it seems that a similar file should be added in this PR: https://github.com/neulab/ExplainaBoard/tree/loaders_support_preprocessor/data/system_outputs/fb15k-237

Reply:
got the answer from the later commits.